psydac/cad/geometry.py

```diff
@@ -326,3 +327,216 @@ def export( self, filename ):
 
         # Close HDF5 file
         h5.close()
+
+#==============================================================================
+def export_geo(filename, patch, periodic=None, comm=None ):
```

Comment:
Please add a good docstring. What is the type of the `patch` argument?

Reply:
done, the type of patch is `igakit.nurbs.NURBS`